account-kit/react/src/index.ts

```diff
@@ -9,6 +9,7 @@ export {
 } from "./createConfig.js";
 export { NoAlchemyAccountContextError } from "./errors.js";
 
+export { AlchemySolanaWeb3Context } from "./AlchemySolanaWeb3Context.js";
```

Comment:
How is this meant to be used? Is it supposed to be used separately from the current alchemy accounts context?

Reply:
It's a context to share the web3 connection.
And if you want to have policy id via context instead of params for useSolanaTransaction

Reply:
Both could be passed in